glusterd2/brick/types.go

```diff
@@ -165,3 +165,13 @@ func (b *Brickinfo) Validate(check InitChecks, allLocalBricks []Brickinfo) error
 	// mandatory check that cannot be skipped forcefully
 	return isBrickInActiveUse(b.Path, allLocalBricks)
 }
+
+//BrickTypeToString converts BrickType to corresponding string
+func (b *Brickinfo) BrickTypeToString() string {
```

Comment:
as we already have a function to convert brick type to string. is it possible to make use of [brick-type](https://github.com/gluster/glusterd2/blob/master/pkg/api/bricktype.go#L14)

Reply:
discard this comment